examples/rvv_strncpy.c

```diff
@@ -0,0 +1,62 @@
+#include "common.h"
+#include <riscv_vector.h>
+#include <string.h>
+
+size_t min(size_t a, size_t b) { return a < b ? a : b; }
+char *strncpy_vec(char *dst, char *src, size_t count) {
+  size_t vlmax = vsetvlmax_e8m8();
```

Comment:
I think maybe having same logic implementation to spec assembly example would be better to see what's different between assembly and intrinsic.